audit/README.md

```diff
@@ -32,16 +32,16 @@ To do this, follow these steps:
 
 1. Create a Terraform workspace (if you haven't already done so) by running
    `terraform workspace new <workspace_name>`
-1. Create a `<workspace_name>.tfvars` file with all of the required
-   variables (see [Inputs](#Inputs) below for details):
+1. Create a `<workspace_name>.tfvars` file with any optional variables
+   that you wish to override (see [Inputs](#Inputs) below for
+   details):
 
    ```console
-   users_account_id = "222222222222"
-
-   admin_usernames = [
-     "first.last",
-     "first2.last2"
-   ]
+   tags = {
+     Team        = "VM Fusion - Development"
+     Application = "COOL - Shared Services Account"
+     Workspace   = "staging"
```

Comment:
Why do some of these README examples have `Workspace   = "staging"` and others have `Workspace   = "production"?`. Shouldn't we just make them all the same?

Reply:
So then maybe we should just make them all consistent with `Workspace = "production"`?

Reply:
I hope you choke on commit 39a66ae sir!

Reply:
Thank you for your support. 😝